### PR TITLE
unquote strings from parameter-file

### PR DESCRIPTION
**Pull request by perbu**
I've made a stab at attempting to unquote the strings from the parameters-file. See #1718. 

I'm a complete Golang newb so style may be way off.

### CODE REVIEW

File: workflow/util/util.go [hunk 1]
```diff
@@ -219,7 +220,12 @@ func SubmitWorkflow(wfIf v1alpha1.WorkflowInterface, wfClientset wfclientset.Int
 			}
 
 			for k, v := range yamlParams {
-				value := fmt.Sprintf("%s", v)
+				// We get quoted strings from the yaml file.
+				value, err := strconv.Unquote(fmt.Sprintf("%s", v))
```
Comment (simster7) [inline workflow/util/util.go]: Very minor suggestion:
```suggestion
				value, err := strconv.Unquote(string(v))
```

File: workflow/util/util.go [hunk 2]
```diff
@@ -219,7 +220,12 @@ func SubmitWorkflow(wfIf v1alpha1.WorkflowInterface, wfClientset wfclientset.Int
...
+				if err != nil {
+					// the string is already clean.
+					value = fmt.Sprintf("%s", v)
```
Comment (simster7) [inline workflow/util/util.go]: Likewise:
```suggestion
					value = string(v)
```

**Comment by Mastergalen** [comment]
The changes seem to be missing in the `2.4.3` release. 

Is it going to be in `2.4.4`?

**Comment by simster7** [comment]
Seems like it wasn't cherry-picked into the last release branch :( This will be in the next release.

**Pull request merged**